oo7-portal-impl/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oo7-portal-impl"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-std = { version = "1.11", features = [ "attributes" ] }
```

Comment:
Now that there is also the `keyring-util` sub-crate, it might be worth considering a switch to workspace so [dependencies](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table) can be maintained in a single place?

Reply:
We discussed this privately, I agree for switching this repository to a workspace but could be done in a separate pull request

Reply:
This is now done, so please make the portal use the workspace feature & rename the project directory to portal. Maybe we should go with `oo7-portal`, `oo7-cli` and so on for now until we decide on something to be used. At least that would be consistent.